api/shareable.py

```diff
@@ -0,0 +1,49 @@
+import os
+from bson.objectid import ObjectId
+from pydantic import BaseModel
+from motor.motor_asyncio import AsyncIOMotorClient
+
+
+class PlanData(BaseModel):
+    json_string: str
+    validator_overrides: list[int]
+
+
+class MongoDBConnection:
+    def __init__(self):
+        url = os.environ.get("PROD_MONGO_URL", "mongodb://localhost:27017")
```

Comment:
Perhaps turn this into a constant named DEFAULT_MONGO_URL and use it throughout?

Reply:
made the change, thanks!